crates/nu-command/Cargo.toml

```diff
@@ -88,6 +88,9 @@ toml = "0.7"
 unicode-segmentation = "1.10"
 ureq = { version = "2.7", default-features = false, features = ["charset", "gzip", "json", "native-tls"] }
 url = "2.2"
+# uu_cp = { path = "../../../coreutils/src/uu/cp"}
+# 0.0.21 not on crates.io yet?
+uu_cp = { git = "https://github.com/uutils/coreutils.git", rev = "862a2df"}
```

Comment:
Does this work for the release that was just published to crates.io?

Reply:
I tried, but apparently it looks like is still not published to `crates.io`. https://crates.io/crates/coreutils/versions

Reply:
Ah I see you noticed this too. We got rate-limited while publishing :) I'll try to publish the rest of the crates tomorrow.